MIGRATION.md

```diff
@@ -750,6 +763,25 @@ Furthermore, the decorator `checkA11y` has been deprecated and renamed to `withA
 
 See the [a11y addon README](https://github.com/storybookjs/storybook/blob/master/addons/a11y/README.md) for more information.
 
+### Addon centered decorator deprecated
```

Comment:
I added a parameter called `layout` and it and 1 of it's possible values is `center`.

This automatically centers the component.

It also supports `fullscreen` and `padded`.

I think we could drop addon-centered entirely in 6.0.0.

Reply:
Yes, let's please move it to the deprecated addons repo, pretty please 🙏  

We can merge this first and follow up separately: https://github.com/storybookjs/storybook/issues/10118